scripts/init.sh

```diff
@@ -77,55 +77,63 @@ if [ -r $DEFAULT ]; then
     source $DEFAULT
 fi
 
-function pidofproc() {
-    if [ $# -ne 3 ]; then
-        echo "Expected three arguments, e.g. $0 -p pidfile daemon-name"
-    fi
+if ! typeset -F pidofproc &>/dev/null; then
```

Comment:
I don't know if `typeset` is supported by Debian's default shell.  Need to test this on potential target platforms.